src/sgame/sg_cmds.cpp

```diff
@@ -4442,6 +4451,58 @@ void Cmd_PrivateMessage_f( gentity_t *ent )
 	}
 }
 
+
+/*
+=================
+Cmd_ReplyPrivateMessage_f
+
+Reply by private message to the last private message sender
+=================
+*/
+
+void Cmd_ReplyPrivateMessage_f(gentity_t *ent)
+{
+	char *msg;
+	int target;
+	bool teamonly = false;
+
+	if (!g_privateMessages.Get() && ent)
+	{
+		ADMP( "\"" N_("Sorry, but private messages have been disabled") "\"" );
+		return;
+	}
+
+	if (trap_Argc() < 2)
+	{
+	ADMP( QQ( N_("usage: /r [message]") ) );
+	return;
+	}
+
+	if ( level.time - ent->client->pers.lastPrivateMessageSenderTime < 3000 )
+	{
+		ADMP( "\"" N_("More than one possible recipient, refusing to send.") "\"" );
+        return;
+	}
+
+	target = ent->client->pers.lastPrivateMessageSender;
+	if (target == -1 || !g_entities[target].client)
```

Comment:
IIRC `client` will always be set so null-checking it is useless. G_SayTo will check whether there is a connected client or not so you could make this test like `if ( target == -1 || !G_SayTo( ... ) )`

Reply:
Is it really necessary? It works very well like this, no need to add more code on this side

Reply:
The benefit of this change would be that if the last sender has disconnected from the server, you would get the "No one to reply to" message, instead of nothing happening.

Reply:
done